.palantir/revapi.yml

```diff
@@ -67,9 +67,45 @@ acceptedBreaks:
       justification: "Removing deprecated method"
   "1.1.0":
     org.apache.iceberg:iceberg-core:
+    - code: "java.class.noLongerImplementsInterface"
+      old: "class org.apache.iceberg.jdbc.JdbcCatalog"
+      new: "class org.apache.iceberg.jdbc.JdbcCatalog"
+      justification: "Needed to remove compilation dependency on Hadoop"
```

Comment:
Since the `setConf(Configuration)` methods are public, I don't think that we can make this change directly. I think we need to add `setConf(Object)` and update to `Configurable<Object>` but leave the `setConf(Configuration)` and deprecate it so we can remove it in 1.3.

That aligns with our published guarantees about compatibility.

Reply:
I updated to leave the old `setConf(Configuration)`